akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -239,8 +240,8 @@ object Behaviors {
    *
    * See also [[akka.actor.typed.Logger.withMdc]]
    */
-  def withMdc[T](staticMdc: Map[String, Any])(behavior: Behavior[T]): Behavior[T] =
-    withMdc[T](staticMdc, (_: T) ⇒ Map.empty[String, Any])(behavior)
+  def withMdcStatic[T](staticMdc: Map[String, Any])(behavior: Behavior[T]): Behavior[T] =
```

Comment:
When we don't use the `T` in the first parameter list using two parameter lists are fine (this is the "normal" style in typed)